test/fsUtils.ts

```diff
@@ -0,0 +1,70 @@
+import { FSUtils } from '../src/helpers/utils/fsUtils';
+import path = require("path");
+import fs = require("fs");
+
+describe('Utils', function () {
+    const pathToCreateEmptyFolder = "test/mock/empty-dir";
+    const pathToNonEmptyDir = "test/mock/non-empty-dir";
+    const dirWithIgnoredFilesPath = "test/mock/empty-dir-with-ignored-files";
+    const dirWithIgnoredVscodePath = "test/mock/empty-dir-with-ignored-vscode";
+
+    before(() => {
+        try {
+            fs.mkdirSync(path.resolve(pathToCreateEmptyFolder));
```

Comment:
Please avoid using `fs.mkdir` and etc. while implementing tests, you need to `stub` needed methods instead.